django_guid/config.py

```diff
@@ -33,6 +34,8 @@ def __init__(self) -> None:
                 raise ImproperlyConfigured('GUID_HEADER_NAME must be a string')  # Note: Case insensitive
             if not isinstance(self.RETURN_HEADER, bool):
                 raise ImproperlyConfigured('RETURN_HEADER must be a boolean')
+            if not isinstance(self.EXPOSE_HEADER, bool):
+                raise ImproperlyConfigured('EXPOSE_HEADER must be a boolean')
```

Comment:
Could we add a check to set `EXPOSE_HEADER` to False if `RETURN_HEADER` is set to false?

Reply:
I can, but the `EXPOSE_HEADER` option is only triggered if `RETURN_HEADER` is `True`. Since it's nested below the `RETURN_HEADER` `if` statement. 

Personally I feel adding logic to toggle it to `False` is unnecessary complexity. 
Also note that there is a test to check that the `EXPOSE_HEADER` does not incorrectly trigger when `RETURN_HEADER` is `False`.

Reply:
You’re right, I totally missed that.